Dockerfile

```diff
@@ -25,6 +25,14 @@ ARG BUILDPLATFORM=${BUILDPLATFORM:-amd64}
 FROM --platform=${BUILDPLATFORM} node:16-slim AS superset-node
 
 ARG NPM_BUILD_CMD="build"
+
+RUN apt-get update -q \
+    && apt-get install -yq --no-install-recommends \
+        python3 \
```

Comment:
Should we pin the `python3` lib here?

Reply:
It's best to pin it for sure, however `python3.9` is not available in the package manager. The alternative is to install python from source, however that takes additional time.

Totally open to alternative solutions as well

Reply:
Understood. I'm fine with this, assuming it's needed to build some NPM libs locally...